demo/src/docker/demo-nodes-docker.yml

```diff
@@ -0,0 +1,24 @@
+-
+  retirementFundUids:
```

Comment:
Is there any realistic way (i.e. simple and quick) to remove this duplication? Would be great if we could provide the demo with one config, and use this to build what's necessary for hub, nodes, and docker

Reply:
:-D